application/features/email.feature

```diff
@@ -205,11 +204,38 @@ Feature: Email
       | to send            | does exist     | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does exist      | delete_mfa   | should NOT  |
       | to send            | does exist     | does exist      | delete_mfa   | should NOT  |
-      | to send            | does NOT exist | does NOT exist  | delete_mfa   | should      |
       | NOT to send        | does NOT exist | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
       | NOT to send        | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
 
+
+  Scenario Outline: When to send an email after a verified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
+      And we are configured to send mfa option removed emails
+      And no mfas exist
+      And a user already exists
+      And a verified u2f mfa option used to exist
```

Comment:
What does it mean that a verified U2F MFA option "used to exist"?

Reply:
It gives the test a way to hold on to a Mfa instance to send to the Emailer class. It doesn't save it to the database and then delete it, since deleting it isn't doable with the external service we use.